orttraining/orttraining/training_api/include/onnxruntime_training_c_api.h

```diff
@@ -144,6 +144,20 @@ struct OrtTrainingApi {
                   size_t inputs_len, _In_reads_(inputs_len) const OrtValue* const* inputs,
                   size_t outputs_len, _Inout_updates_all_(outputs_len) OrtValue** outputs);
 
+  /** \brief Set the learning rate for this training session.
+  *
+  * This function allows users to set the learning rate for the training session. The training
+  * session by default starts with a learning rate of 0.001. It can be overwritten by invoking
+  * this function with the desired learning rate.
+  *
+  * \param[in] sess The training session on which the learning rate needs to be set.
+  * \param[in] learning_rate Desired learning rate to set.
+  *
+  * \snippet{doc} snippets.dox OrtStatus Return Value
+  *
+  */
+  ORT_API2_STATUS(SetLearningRate, _Inout_ OrtTrainingSession* sess, _In_ float learning_rate);
```

Comment:
I think it's better we call it InitialLearningRate?

Reply:
Do we envision users building their own lr scheduler and periodically invoking `SetLearningRate`? If yes, we should probably call it `SetLearningRate`. If we think this function should only be called to set the initial learning rate, we could then call it `InitialLearningRate`. What do you think?

Reply:
I see, thanks for the explanation! I think this is fine, let's keep the name.